README.md

```diff
@@ -25,7 +25,7 @@ A design system and [React](https://facebook.github.io/react/) component library
 Mineral UI is distributed as a multitude of [npm packages](https://www.npmjs.com/search?q=@mineral-ui). E.g., to install the Hello and World packages:
 
 ```sh
-npm install --save @mineral-ui/hello @mineral-ui/world
+npm install --save @mineral-ui/style-utils @mineral-ui/hello @mineral-ui/world
```

Comment:
This should include an example of installing peer dependencies.  We should also verify any needed updates to individual component docs.